README.md

```diff
@@ -27,6 +27,23 @@ $> docker run --name=ntp             \
               cturra/ntp
 ```
 
+Load your own NTP config file into the container
+---
+
+Add 
+
+```-v /path/to/folder/containing/my_config_file:/data```
+
+to the command above. 
+
+
+Note that this way, one can also save the NTP drift file accross container restarts by adding 
```

Comment:
the ntp drift file is kept across container restarts. it would be lost with a redeploy (stop -> rm -> run) however. this is not something i'd personally use a volume to keep since it can be regenerated without lost functionality -- so can be ephemeral. 

```
$> docker exec ntp cat /var/db/ntpd.drift
-6.373
$> docker restart ntp
ntp
$> docker exec ntp cat /var/db/ntpd.drift
-6.373
```